cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -837,10 +837,13 @@ public Tracing getTracing() {
      * Creates the ClusterRoleBinding which is used to bind the Kafka Connect SA to the ClusterRole
      * which permissions the Kafka init container to access K8S nodes (necessary for rack-awareness).
      *
+     * @param rack Rack configuration.
+     * @param name ClusterRoleBinding name.
+     *
      * @return The cluster role binding.
      */
-    public ClusterRoleBinding generateClusterRoleBinding() {
-        if (rack == null) {
+    public ClusterRoleBinding generateClusterRoleBinding(Rack rack, String name) {
+        if (rack == null || name == null || name.trim().isEmpty()) {
```

Comment:
I think passing the rack explicitly is clearer. I'll remove the name that is not needed.

Reply:
Why is it clearer when it is part of the model object to which this method belongs anyway? That is why it was designed around the class hierarchy to use these things internally. Without it it can be all rewritten as static methods and everything can be passwed around in parameters.

Reply:
Right, I see it now. Let mi fix that.